build/ci/save_driver_pods_logs.sh

```diff
@@ -1,9 +1,19 @@
 #!/bin/bash -x
 
-kubectl logs $(kubectl get pod -l csi | grep controller | awk '{print$1}') -c ibm-block-csi-controller > /tmp/driver_$(kubectl get pod -l csi | grep controller | awk '{print$1}')_logs.txt
-kubectl logs $(kubectl get pod -l csi | grep node | awk '{print$1}') -c ibm-block-csi-node > /tmp/driver_$(kubectl get pod -l csi | grep node | awk '{print$1}')_logs.txt
-kubectl logs $(kubectl get pod -l csi | grep operator | awk '{print$1}') > /tmp/driver_$(kubectl get pod -l csi | grep operator | awk '{print$1}')_logs.txt
+get_pod_logs_or_events (){
```

Comment:
I suggest renaming the function to `run_action_on_pod` since it can run every action.

Reply:
done

Reply:
done